### DISCUSSION TITLE
block data for gouging fire and raging bolt

### DISCUSSION THREAD

**Discussion by arcenedon** [kwsch/PKHeX#4127] [Q&A]
hi! i was trying to look for the block data for the gouging fire and raging bolt encounters, since mine aren't appearing in game. is there anyone that knows what they are so i can check them? thanks!

**Comment by kwsch** [comment]
Briefly checking the SaveBlockAccessor9SV list of blocks, there does not appear to be any block listed related to them, however...

They're the two here:
`WEVT_S2_SIDE02_0010_STATE`
`WEVT_S2_SIDE02_0020_STATE`

Try changing those.

**Reply by arcenedon** [reply]
it worked! changed the value's to 0 and they appeared thankfully. thank you for your help!

**Reply by Syntex93** [reply]
> Briefly checking the SaveBlockAccessor9SV list of blocks, there does not appear to be any block listed related to them, however...
> 
> They're the two here: `WEVT_S2_SIDE02_0010_STATE` `WEVT_S2_SIDE02_0020_STATE`
> 
> Try changing those.

Hi I was trying to follow what you said before but i can't find

WEVT_S2_SIDE02_0010_STATE & WEVT_S2_SIDE02_0020_STATE Rows.

I checked all the list multiple times but nothing

Do you know which one to modify to respawn ogerpon too eventually?

Thanks in advance for your help

**Reply by Lusamine** [reply]
They've been renamed since this discussion (which is from 2023). Type "paradox" and hit enter to search for all names that include "paradox".
![image](https://github.com/user-attachments/assets/907df03b-98c3-42c8-81fd-93ce98244953)

> Do you know which one to modify to respawn ogerpon too eventually?

Ogerpon is not a single setting to enable because it's heavily tied to story, and since the game is open world, story status is stored in hundreds of tiny save blocks. You're probably better off playing through the story if you don't know how to research every single change that's needed.
